docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Documentation Instructions
+
+To prepare the local docs for preview:
+
+```bash
+$ python -m venv env
+$ source env/bin/activate
+$ pip install -r requirements.txt
+$ `cd $(find env/lib -type d -name sphinxcontrib) && wget https://raw.githubusercontent.com/wpilibsuite/sphinxext-remoteliteralinclude/main/sphinxext/remoteliteralinclude.py`
```

Comment:
This happens to work, but is not what is intended.  The command substitution ``` `cmd` ``` creates a new shell and then evaluates its stdout as bash. In this case the only output is to stderr so nothing gets executed. Using just a subshell `(cmd)` is what is intended (to not make the `cd` change the current shell).  Better yet would be to use `wget`'s `--directory-prefix` or `-P` to set the output directory which eliminates the `cd` and hence the need for a subshell:

```shell
wget -P $(find env/lib -type d -name sphinxcontrib) https://raw.githubusercontent.com/wpilibsuite/sphinxext-remoteliteralinclude/main/sphinxext/remoteliteralinclude.py 
```

Reply:
Oh, I didn't know that wget could do that. Guess I should read the man pages more often. Fixed.